app/assets/stylesheets/ama_layout/layout_components/sections.scss

```diff
@@ -104,12 +104,15 @@
 .section-two-thirds{
   @extend .section;
   @include large-8;
-  @include grid-column-end;
```

Comment:
I'm actually not sure if we can take this out, I think you're ok to leave it, because your float:none below should override it on mobile?